src/structured-text/converters/convert_string_value.cpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************\
+
+Module: Structured Text Language Conversion
+
+Author: Matthias Weiss, matthias.weiss@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Structured Text Language Conversion
+
+#include "convert_string_value.h"
+
+exprt convert_identifier(const std::string &src)
+{
+  // Cut the leading and trailing quotes
+  std::string value = src.substr(1, src.length() - 2);
+
+  typet type(ID_structured_text_identifier);
+
+  exprt result(value, type);
+  return result;
+}
+
+exprt convert_title(const std::string &src)
+{
+  std::string value(src);
+  typet type(ID_structured_text_title);
+
+  exprt result(value, type);
+  return result;
+}
+
+exprt convert_label(const std::string &src)
```

Comment:
Ideally we use a `code_labelt` here and eventually merge it with the subsequent `codet` - although that can happen at `typecheck`.